src/script/assets/AssetMapper.ts

```diff
@@ -20,12 +20,9 @@
 import {Asset} from '../entity/message/Asset';
 import {AssetRemoteData} from './AssetRemoteData';
 
-export interface MappedAsset {
-  medium?: AssetRemoteData;
-  preview?: AssetRemoteData;
-}
+export type MappedAsset = {[index: string]: AssetRemoteData};
```

Comment:
Why loosening this one up?

Reply:
Because there is `medium`, `preview` and `complete` already and it's not clear yet if there are any others too so I didn't want to limit the possibilities.

Reply:
Hint: next time use `Record<string, AssetRemoteData>` :sunglasses:

Reply:
cool one, indeed :)